### PR TITLE
N14: BUGOUT blurb

### PR DESCRIPTION
**Pull request by Terkwood**
This is the blurb for BUGOUT in newsletter 14.

_Part of #278 ("Newsletter 14")_

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -62,6 +62,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### BUGOUT
+
+![Play Go against AI and friends on the web](BUGOUT.jpg) _Playing KataGo AI in 9x9_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:67]: Captions should go on a new line:
```suggestion
![Play Go against AI and friends on the web](BUGOUT.jpg)
_Playing KataGo AI in 9x9_
```

File: content/posts/newsletter-014/index.md [hunk 2]
```diff
@@ -62,6 +62,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[BUGOUT](https://github.com/Terkwood/BUGOUT) is a web application
+which allows you to play Go/Baduk/Weiqi against a leading AI
+([KataGo](https://github.com/lightvector/KataGo)).  It provides a
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:71]:
```suggestion
([KataGo](https://github.com/lightvector/KataGo)). It provides a
```

File: content/posts/newsletter-014/index.md [hunk 3]
```diff
@@ -62,6 +62,26 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+multiplayer mode so that you can play other humans, either by joining a
+public queue or sharing a private URL to your friend.
+
+The user interface is lifted from [Sabaki](https://github.com/SabakiHQ/Sabaki).
+
+The initial installation's AI is powered by an energy-efficient
+[dev board](https://developer.nvidia.com/embedded/jetson-nano-developer-kit).
+
+BUGOUT is marching actively toward production, at which point the
+team will publish the website address and invite users.  The author
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:81]:
```suggestion
team will publish the website address and invite users. The author
```

**Review by 17cupsofcoffee — APPROVED** [review]
A few little tweaks needed, but the content looks okay. Not sure if @ozkriff might prefer to switch the inline links out for reference-style links? I'm happy either way.

**Comment by Terkwood** [comment]
Thank you

**Review by ozkriff — APPROVED** [review]
:+1:  (made a few fmt tweaks)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**